src/sources/ffmpeg.rs

```diff
@@ -0,0 +1,43 @@
+use songbird::input::{
+    error::{Error, Result},
+    Codec, Container, Input, Metadata, Reader,
+};
+use std::process::{Child, Command, Stdio};
+
+pub async fn ffmpeg_yt(mut yt: Child, metadata: Metadata, pre_args: &[&str]) -> Result<Input> {
+    let ffmpeg_args = [
+        "-i",
+        "-", // read from stdout
+        "-f",
+        "s16le", // use PCM signed 16-bit little-endian format
+        "-ac",
+        "2", // set two audio channels
+        "-ar",
+        "48000", // set audio sample rate of 48000Hz
+        "-acodec",
+        "pcm_f32le", // set pcm_f32le codec
```

Comment:
This comment adds nothing.

Reply:
I disagree here. These are arguments for FFMPEG, if you want to know about the FFMPEG codecs, there is a FFMPEG documentation website that you can look, the comment already says everything we need to know. At best I'd say we could add `audio`, aka `// set pcm_f32le audio codec`, since -acodec automatically scopes to Audio streams.